ci-operator/config/openshift-assisted/bug-master-bot/openshift-assisted-bug-master-bot-main.yaml

```diff
@@ -0,0 +1,25 @@
+binary_build_commands: make install
+build_root:
+  project_image:
+    dockerfile_literal: |-
+      FROM registry.ci.openshift.org/ocp/ubi-python-39:8
+      USER 0:0
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: |
+  make full-install
+tests:
```

Comment:
Do you mean to add a test for the image ?

Reply:
something like: https://github.com/openshift/release/blob/1a3967f583b993c0fc59779fbac562a6258736c8/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L100-L105

so that there's an ``images`` job that builds that image

Reply:
Do you think there is an advantage this way? we currently just want to format the code

Reply:
not sure I understand
do you ask if there's an advantage to validating that the Dockerfile is buildable in PRs?

Reply:
No, AFAIK the part you linked as about building the images that will be used later, rather then actually a job that tests building the images, although I guess that is the effect indirectly, but it would be easier to just add a test that tries to build the image no ?

Reply:
No, in prow you cannot do dind so it won't work (unless you're acquiring an equinix machine or take a cluster from a pool. but it uses much more resources)

Reply:
Thanks